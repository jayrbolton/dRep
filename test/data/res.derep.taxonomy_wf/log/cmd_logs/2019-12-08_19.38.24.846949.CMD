/usr/local/bin/mash paste /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/MASH_files/sketches/chunk_0/chunk_all.msh /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/MASH_files/sketches/chunk_0/Bin.018.fasta.msh /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/MASH_files/sketches/chunk_0/Bin.012.fasta.msh /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/MASH_files/sketches/chunk_0/Bin.039.fasta.msh /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/MASH_files/sketches/chunk_0/Bin.034.fasta.msh /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/MASH_files/sketches/chunk_0/Bin.040.fasta.msh /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/MASH_files/sketches/chunk_0/Bin.044.fasta.msh /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/MASH_files/sketches/chunk_0/Bin.027.fasta.msh /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/MASH_files/sketches/chunk_0/Bin.038.fasta.msh
