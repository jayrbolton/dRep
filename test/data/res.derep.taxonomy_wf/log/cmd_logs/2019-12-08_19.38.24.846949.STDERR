Writing /home/sumin/kbsdk-workspace/dRep/test/data/res.derep.taxonomy_wf/data/MASH_files/sketches/chunk_0/chunk_all.msh...
